docs/en/guides/57-data-management/04-data-recycle.md

```diff
@@ -12,6 +12,54 @@ There are two types of data:
 
 If the data size is significant, you can run several commands ([Enterprise Edition Features](/guides/products/dee/enterprise-features)) to delete these data and free up storage space.
 
+## Spill Data Storage
+
+Self-hosted Databend supports spilling intermediate query results to disk when memory usage exceeds available limits. Users can configure where spill data is stored, choosing between local disk storage and a remote S3-compatible bucket.
+
+### Spill Storage Options
+
+Databend provides the following spill storage configurations:
+
+- Local Disk Storage: Spilled data is written to a specified local directory in the query node (Windows only).
```

Comment:
only window function（https://docs.databend.com/sql/sql-functions/window-functions/ ） for now